doc/lua_api.txt

```diff
@@ -2398,13 +2425,14 @@ This is basically a reference to a C++ `ServerActiveObject`
 * `override_day_night_ratio(ratio or nil)`
     * `0`...`1`: Overrides day-night ratio, controlling sunlight to a specific amount
     * `nil`: Disables override, defaulting to sunlight based on day-night cycle
-* `set_local_animation({x=0, y=79}, {x=168, y=187}, {x=189, y=198}, {x=200, y=219}, frame_speed=30)`:
-  set animation for player model in third person view
-    * stand/idle animation key frames
-    * walk animation key frames
-    * dig animation key frames
-    * walk+dig animation key frames
-    * animation frame speed
+* set animation for player model in third person view
+
+        set_local_animation({x=0, y=79}, -- < stand/idle animation key frames
+            {x=168, y=187}, -- < walk animation key frames
+            {x=189, y=198}, -- <  dig animation key frames
+            {x=200, y=219}, -- <  walk+dig animation key frames
+            frame_speed=30): -- <  animation frame speed
+ ^
```

Comment:
The function name and params should be on the first line, having an example below doesn't replace that.

Still has a superfluous `^`.